Sources/Public/MediaConstants+Public.swift

```diff
@@ -96,7 +96,6 @@ public class MediaConstants: NSObject {
     public class MediaObjectKey: NSObject {
         public static let RESUMED = "media.resumed"
         public static let PREROLL_TRACKING_WAITING_TIME = "media.prerollwaitingtime"
-        public static let GRANULAR_AD_TRACKING = "media.granularadtracking"
```

Comment:
This is a public API constant. It is possible customers may still use this when migrating to Edge Media.

Reply:
We have a path to migrate this using the new trackerConfig. Should we still keep this?

Reply:
This is covered in the migration guide - https://github.com/adobe/aepsdk-edgemedia-ios/blob/dev/Documentation/migration-guide.md